<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configDialog</class>
 <widget class="QDialog" name="configDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="znotes.qrc">
    <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Common</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Path to notes directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ed_NotesPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btn_NotesPath">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="formAlignment">
          <set>Qt::AlignJustify|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QCheckBox" name="cb_HideStart">
           <property name="text">
            <string>Hide in tray on start</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="cb_ToolbarHide">
           <property name="text">
            <string>Hide toolbar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_FrameHide">
           <property name="text">
            <string>Hide window's frame</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_StayTop">
           <property name="text">
            <string>Stay on top</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Font:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lb_FontExample">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>Eat more of these soft French loaves and drink tea.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_FontChange">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Change</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Scripts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tabScripts">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_ScriptAdd">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="znotes.qrc">
             <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ScriptRemove">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="znotes.qrc">
             <normaloff>:/res/remove.png</normaloff>:/res/remove.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cb_ScriptShowOutput">
           <property name="text">
            <string>Show script's output</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_ScriptCopyOutput">
           <property name="text">
            <string>Copy script's output to clipboard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Toolbar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="cb_tbHideEdit">
         <property name="text">
          <string>Hide edit section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/add.png</normaloff>:/res/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_tbHideMove">
         <property name="text">
          <string>Hide move section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/next.png</normaloff>:/res/next.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_tbHideCopy">
         <property name="text">
          <string>Hide copy section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/copy.png</normaloff>:/res/copy.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_tbHideSetup">
         <property name="text">
          <string>Hide setup section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/settings.png</normaloff>:/res/settings.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_tbHideRun">
         <property name="text">
          <string>Hide run section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/exec.png</normaloff>:/res/exec.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_tbHideExit">
         <property name="text">
          <string>Hide exit section</string>
         </property>
         <property name="icon">
          <iconset resource="znotes.qrc">
           <normaloff>:/res/exit.png</normaloff>:/res/exit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="znotes.qrc"/>
 </resources>
 <connections/>
</ui>
